From acc58ce84aaacc7abb2a9d20712b0c40a8b27ae2 Mon Sep 17 00:00:00 2001
From: timsong-cpp <rs2740@gmail.com>
Date: Sun, 11 Dec 2016 07:08:56 -0500
Subject: [PATCH] Revert "[macros] In range, change mandatory space followed by
 linebreak opportunity into normal space with discouraged linebreak. This
 fixes the problem that mandatory spaces appeared at the end of the line,
 tearing up the column."

This reverts commit 78e1435bbe5d8ed46e3cd69975c0b666da625a32.
---
 source/macros.tex | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/source/macros.tex b/source/macros.tex
index aacbf0d..d3c7aa2 100644
--- a/source/macros.tex
+++ b/source/macros.tex
@@ -282,7 +282,7 @@
 \newcommand{\mname}[1]{\tcode{\unun\ungap#1\ungap\unun}}
 
 %% Ranges
-\newcommand{\Range}[4]{\tcode{#1#3,\penalty2000{} #4#2}\xspace}
+\newcommand{\Range}[4]{\tcode{#1#3,~\brk{}#4#2}\xspace}
 \newcommand{\crange}[2]{\Range{[}{]}{#1}{#2}}
 \newcommand{\brange}[2]{\Range{(}{]}{#1}{#2}}
 \newcommand{\orange}[2]{\Range{(}{)}{#1}{#2}}
-- 
2.7.4

